index.html

```diff
@@ -2322,6 +2258,169 @@ <h2>JSON-LD</h2>
 
       </section>
 
+        <section>
+            <h2>JSON Web Token</h2>
+
+            <p>
+                The verifiable credential data model as described in Section <a href="#core-data-model"></a> can be
+                encoded as a JSON Web Token (JWT) [[!RFC7519]] by doing the following:
+            </p>
+
+            <ul>
+                <li>The signature of the JWT is produced by the issuer of the verifiable credential.</li>
+                <li>Because JWS implicitly provides a verifiable proof of the contained verifiable credentials, the "proof" is no longer needed. Please note that
+                    the "jti" claim represents a nonce value.</li>
```

Comment:
There are two ways that we could go with this: Add the JWT keywords (jti, iat, etc.) to the JSON-LD Credentials vocab... or leave them out, and move the JSON-LD Context down one level.

The question this raises is: Is the top level just an encapsulating container? Or is it a VC? If it's  just a container, where does evidence and termsOfUse go? I expect we'll end up with the top level being a VC, which then begs the question of whether or not JSON-LD processing is required or not during verification to ensure the semantic mappings between the sending and receiving application are checked.